PostSharp.Community.StructuralEquality/StructuralEqualityAttribute.cs

```diff
@@ -40,5 +40,9 @@ public class StructuralEqualityAttribute : MulticastAttribute
         /// </summary>
         public bool IgnoreBaseClass { get; set; }
 
+        /// <summary>
+        /// If true, equality operators are neither checked nor replaced.
+        /// </summary>
+        public bool DoNotAddEqualityOperators { get; set; }
```

Comment:
I considered not porting this property and going with "if you don't add equality operators, they won't be added; i.e. by default, they won't be added". But I understand that users might be surprised when equality operators aren't created for them automatically.

Reply:
We decided offline to keep it like this for now.